src/vscode-bicep/schemas/bicepconfig.schema.json

```diff
@@ -697,9 +697,16 @@
       "description": "The directory in which Bicep may cache templates downloaded from remote registries"
     },
     "experimentalFeaturesEnabled": {
+      "_comment": "Documentation for these features can be found at https://github.com/Azure/bicep/blob/main/docs/experimental-features.md",
       "type": "object",
       "description": "The experimental Bicep features that should be enabled or disabled for templates using this configuration file",
       "properties": {
+        "microsoftGraphPreview": {
+          "type": "boolean"
+        },
+        "assertions": {
```

Comment:
see relevant comment here https://github.com/Azure/bicep/issues/13109#issuecomment-1912733643. I am adding this because it is the value that is used in the default configuration. That said, there is already a similar property `asserts` that maybe should be removed. The only reason for me to add this is so that I can add the properties for provider configuration starting from a valid state.

Reply:
Could you remove `asserts`, too? It looks like [only `assertions` is used](https://github.com/Azure/bicep/blob/4d77843b632c58d5c942c1c44329385a384ba1a9/src/Bicep.Core/Configuration/ExperimentalFeaturesEnabled.cs#L18).